src/BaseApiAbstract.php

```diff
@@ -316,7 +316,7 @@ protected function getDefaultRequestData($parametersType, $parameters, $auth = t
                 foreach ($parameters as $key => $value) {
                     if (is_array($value)) {
                         foreach ($value as $item) {
-                            $queryStringParts[] = "$key" . "[]=$item";
+                            $queryStringParts[] = "$key=$item";
```

Comment:
I think we must url encode `$item` here (and in line below and `getDefaultRequestData()`).
Can you check how URL will look if `$item="123:/content/page.html#anchor?param1=!&param2=*"`? I expect something like
```
mykey=123%3A%2Fcontent%2Fpage.html%23anchor%3Fparam1%3D%21%26param2%3D%2A
```

Reply:
https://github.com/Smartling/api-sdk-php/pull/118